statistics/statistics.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Rubik&display=swap" rel="stylesheet" />
+    <link href="statistics.css" rel="stylesheet" />
+    <title>Statistics page</title>
+</head>
+<body>
+    <header class="header">
+        <fiqure class="header-img">
+            <img src="/images/corona3.png"  alt="Header corona image">
+        </fiqure>
+        <section class="header-locatin ">
+            <img src="/images/loction.png" width="20px"  alt="">
+            <h1 class="country">
+                Palestine
+            </h1> 
```

Comment:
use `<figcaption>` instead of `<h1>` or `<h2>` or `<h3>` or ... when it is related to an image  `<img>`
And give it your right style